### ISSUE TITLE
Unrecognized architecture Darwin-arm64

### ISSUE THREAD

**Issue by dcblack** [romkatv/gitstatus#286]
Apple Silicon M1X on Monterey 12.1 reports "WARNING: Unrecognized architecture Darwin-arm64 - taking default".

**Comment by romkatv** [comment]
Which application prints this warning and when?

**Comment by dcblack** [comment]
Good question. So far I have not been able to track it down. I was suspecting gitstatus, but I am unsure. If I knew the path to find the executable, I might do some testing.

Note: MacBook Pro with Apple Silicon running latest Monterey 12.1. I'm a hardware/software engineer (chip design/verification and RTOS/bare-metal development mostly in C++).

**Comment by romkatv** [comment]
Is this warning printed in a terminal?

When is it printed? In other words, what do you need to do to have this warning printed?

Is it printed when you type `zsh` and press ENTER in a terminal?

Is it printed when you type `zsh -f` and press ENTER in a terminal?

**Comment by dcblack** [comment]
Sorry for the slow turn-around.

The warning is printed on the terminal, and I have found the cause. It was something not related to p10k or zsh. It turns out we use a something called modulecmd (see http://modules.sourceforge.net/) and it came from there. I am updating the warning message to include the source in the rather generic message.

The reason I thought the problem lay here was several fold:
1. I had just updated to Apple Silicon
2. p10k was issuing a warning about the message; therefore, I figured it came from somewhere inside.
3. A quick grep of dotfiles in my home directory did not reveal the culprit (which should have been a clue that it was not zsh).
4. I noticed gitstatus for some reason I don't recall

I am closing this false alarm.
